strategies/aartur/preparation.py

```diff
@@ -0,0 +1,104 @@
+from rules import State, Wall, Player
+
+
+class AdvancedState:
+    """More convenient State, containing current player and other players."""
+
+    def __init__(self, state: State, player_name: str):
+        self.cells = state.cells
+        self.size_y = len(self.cells)
+        self.size_x = len(self.cells[0])
+
+        self.player: Player
+        self.other_players: list[Player] = []
+        for y in range(self.size_y):
+            for x in range(self.size_x):
+                cell = self.cells[y][x]
```

Comment:
get_cell

Reply:
I will fix it with other minor fixes (don't wanna do the whole commit for that thing now)